python/composio/tools/base/runtime.py

```diff
@@ -253,12 +253,23 @@ def _parse_docstring(
     return header, params, returns
 
 
+def _get_connected_account(app: str, entity_id: str) -> t.Optional[ConnectedAccountModel]:
```

Comment:
Consider adding docstring to the new `_get_connected_account` function to maintain consistency with codebase documentation standards. The docstring should describe parameters, return type, and possible exceptions.